docs/buildkitd.toml.md

```diff
@@ -29,6 +29,19 @@ insecure-entitlements = [ "network.host", "security.insecure" ]
   options=["edns0"]
   searchDomains=["example.com"]
 
+# If allowed-frontends is empty, all frontends (e.g, "dockerfile.v0"
+# and "gateway.v0") are allowed.  Otherwise, only those listed are
+# allowed
+allowed-frontends = []
+
+# If allowed-gateway-sources is empty, all gateway sources are allowed.
+# Otherwise, only sources that match the patterns in this list will be
+# allowed.  Patterns are matched using <https://pkg.go.dev/github.com/bmatcuk/doublestar/v4#Match>.
```

Comment:
Can we use https://github.com/moby/buildkit/tree/master/util/wildcard

Reply:
Sure.  I'll make the changes.